### ISSUE TITLE
Routing Issue

### ISSUE THREAD

**Issue by Nendra** [Warsaalk/trashsim-public#2]
I have setup everything and solve some issue on the way, but now I'm stuck with this error.

```
[26-Dec-2023 11:07:56 Europe/Berlin] PHP Fatal error:  Uncaught Plinth\Exception\PlinthException: You can only define 1 default route in /var/www/xxxxx/public_html/vendor/plinth/plinth/src/Plinth/Routing/Router.php:57
Stack trace:
#0 /var/www/xxxxx/public_html/vendor/plinth/plinth/src/Plinth/Main.php(394): Plinth\Routing\Router->loadRoutes()
#1 /var/www/xxxxx/public_html/vendor/plinth/plinth/src/Plinth/Main.php(363): Plinth\Main->handleRouter()
#2 /var/www/xxxxx/public_html/vendor/plinth/plinth/src/Plinth/Main.php(410): Plinth\Main->executeHandlers()
#3 /var/www/xxxxx/public_html/public/index.php(11): Plinth\Main->handleRequest()
#4 {main}
  thrown in /var/www/xxxxx/public_html/vendor/plinth/plinth/src/Plinth/Routing/Router.php on line 57
```

and when I disable a route in the app config, I just ending in a Hello World Page.

**Comment by stevecohenfr** [comment]
Same for me. After debugging i'm stuck on "Hello World". I thing AngularJS is not loading

**Comment by stevecohenfr** [comment]
I wanted to learn how to use Plinth but it's his own undocumented mini php framework 😅
https://github.com/Warsaalk/Plinth

**Comment by klaasvp** [comment]
Hi :) 

The composer file still contained an older version of Plinth.
I've now update that version, which shouldn't overwrite the affected files anymore.
Commit e8f1acf335ab3abd23f1cfc53f26262fdfdaac0b fixes it.

Just delete the whole directory on your computer and clone the repository again.

Plinth development stopped a while ago, the plans for documentation were on the roadmap.
I also used it in project in my job but we switched to NodeJS so there was no need anymore for it.

Best regards,
Klaas
